activesupport/lib/active_support/backtrace_cleaner.rb

```diff
@@ -91,7 +91,7 @@ def add_gem_filter
         gems_paths = (Gem.path | [Gem.default_dir]).map { |p| Regexp.escape(p) }
```

Comment:
I was concerned about relative paths in `Gem.path`, but it looks like they're always expanded:

```bash
$ GEM_PATH=relative_gems ruby -e 'p Gem.path'
["/Users/eugene/test/relative_gems", "/Users/eugene/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0"]
```

https://github.com/rubygems/rubygems/blob/v3.1.4/lib/rubygems/path_support.rb#L71